src/models/User.js

```diff
@@ -45,8 +55,11 @@ module.exports = (sequelize, DataTypes) => {
     },
     {}
   );
-  User.associate = () => {
-    // add model as function parameter and define associations here
+  User.associate = (models) => {
+    User.belongsTo(models.Supplier, {
+      foreignKey: 'supplierId',
+      as: 'affiliateSupplier'
+    });
   };
   return User;
 };
```

Comment:
Consider renaming this file too from `User.js` to `user.js` to be uniform with others

Reply:
It's an autogenerated file. By sequelize cli @emmanuelnwankwo . I think we should leave it be

Reply:
That's okay